packages/amplify-graphql-directives/src/directives/belongs-to.ts

```diff
@@ -1,8 +1,9 @@
 import { Directive } from './directive';
 
 const name = 'belongsTo';
+// TODO: review overrideIndexName before merging to main
```

Comment:
Is `override` prefix necessary here? Can we just keep it `indexName` and if specified it means an override, otherwise use default?

Reply:
We will discuss further in API naming review, but I wouldn't favor overloading the `indexName` directive with different behavior like that.